x-pack/plugins/maps/public/connected_components/timeslider/timeslider.tsx

```diff
@@ -59,29 +51,16 @@ export class Timeslider extends Component<Props, {}> {
     }
 
     this._subscriptions.add(
-      controlGroup
-        .getOutput$()
-        .pipe(
-          distinctUntilChanged(({ timeslice: timesliceA }, { timeslice: timesliceB }) =>
-            _.isEqual(timesliceA, timesliceB)
-          )
-        )
-        .subscribe(({ timeslice }) => {
-          // use waitForTimesliceToLoad$ observable to wait until next frame loaded
-          // .pipe(first()) waits until the first value is emitted from an observable and then automatically unsubscribes
-          this.props.waitForTimesliceToLoad$.pipe(first()).subscribe(() => {
```

Comment:
We have lost `waitForTimesliceToLoad$` and waiting for layers to load before going to next timeslice. In the new system, this is done by controlGroupApi.parent.dataLoading observable. `ControlGroupRenderer` needs to accept another parameter `dataLoading`. Then, in this usage, `dataLoading` needs to be set to true when `timeslice$` emits and set to false when `this.props.waitForTimesliceToLoad$.pipe(first())` emits

Reply:
Done in 5c5e575c32a2454471e0b49dedefb9526cd6d05a :+1: